ai-platform-unified/notebooks/official/automl/automl-text-classification.ipynb

```diff
@@ -0,0 +1,1413 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    job = jobs.BatchPredictionJob(batch_prediction_job_name)
</pre><p> </p><p>Alternatively, you could do:</p><p> </p><pre class="ql-syntax" spellcheck="false">batch_prediction_job = model.batch_predict(...)
</pre><p> </p><p>Just another option.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/360/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>

Reply:
<p>Ack. I think it's useful to show customers how to instantiate the <code>BatchPredictionJob</code>  class, since they may need to do so in a production environment (where calls to <code>batch_predict()</code>  and retrieving the batch prediction results might run in separate processes).</p>